client/src/components/credentials/credentials.js

```diff
@@ -370,29 +282,25 @@ class Credentials extends React.Component {
     }
 
     if (credentials && credentials.length) {
-      this.sortCredentials(filteredItems);
-
       return (
         <React.Fragment>
           <div className="quipucords-view-container">
             <ViewToolbar
               viewType={viewTypes.CREDENTIALS_VIEW}
-              totalCount={credentials.length}
-              filteredCount={filteredItems.length}
               filterFields={CredentialFilterFields}
               sortFields={CredentialSortFields}
               actions={this.renderActions()}
```

Comment:
Optional: At some point, for readability, we should probably rename these to correspond to the component, i.e. ```this.renderToolbarActions()```